client/signup/utils.js

```diff
@@ -175,7 +175,7 @@ export function getDesignTypeForSiteGoals( siteGoals, flow ) {
 	return 'blog';
 }
 
-export function getFilteredSteps( flowName, progress ) {
+function getFilteredSteps( flowName, progress ) {
```

Comment:
It makes sense to remove the export in this PR, but I noticed that in @spen's work on the progress store, he's using `getFilteredSteps` in a few places, so it might be worth keeping the `export` around for a bit longer? https://github.com/Automattic/wp-calypso/pull/34885

Reply:
Thanks for the heads up! No point making the #34885 merge harder than it needs to be.